rapl-daemon/util_test.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+	"math"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var raplDir string
+
+const maxWattage uint64 = 1500000
+
+func TestMain(m *testing.M) {
+	var err error
+	raplDir, err = ioutil.TempDir("", raplPrefixCPU)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer os.RemoveAll(raplDir)
+
+	// Create temporary directory that mocks powercap subsytem
+	zonePath := filepath.Join(raplDir, raplPrefixCPU+":0")
+	err = os.Mkdir(zonePath, 755)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	initialWatts := strconv.FormatUint(maxWattage, 10)
+
+	err = ioutil.WriteFile(filepath.Join(zonePath, maxPowerFileShortWindow), []byte(initialWatts), 0444)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	err = ioutil.WriteFile(filepath.Join(zonePath, powerLimitFileShortWindow), []byte(initialWatts), 0644)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	os.Exit(m.Run())
+}
+
+// TODO(rdelvalle): Create filesystem only once and allow tests to use it
+func TestCapNode(t *testing.T) {
+	err := capNode(raplDir, 95)
+	assert.NoError(t, err)
+
+	t.Run("badPercentage", func(t *testing.T) {
```

Comment:
I think it's safe to use spaces ("bad percentage") as the words are hyphenated by default.
Although, I do not mind the camelcasing.

Reply:
Good point, fixed.